test/bdd/features/get-errors.feature

```diff
@@ -7,20 +7,7 @@ Feature: Get errors test
   Scenario: Getting non existent UAL
     Given I setup 4 nodes
     And I wait for 2 seconds
-    And I call get directly to ot-node 1 with nonExistentUAL
-    And I wait for last resolve to finalize
-    Then Last Get operation finished with status: GetAssertionIdError
-
-  #@get-errors
-  #Scenario: GET operation result on a node with minimum replication factor greater than the number of nodes
-    #Given I setup 4 nodes
-    #And I wait for 2 seconds
-    #And I call publish on node 1 with validAssertion
-    #Then Last PUBLISH operation finished with status: COMPLETED
-    #When I setup node 5 with minimumAckResponses.get set to 10
-    #And I wait for 2 seconds
-    #And I get operation result from node 5 for last published assertion
-    #And I wait for last resolve to finalize
-    #Then Last GET operation finished with status: GetNetworkError
-
-
+    
+    When I call Get directly on the node 1 with nonExistentUAL
```

Comment:
Can you please add more tests for get historical, we are currently covering only happy path